app/views/manual_sections/show.html.erb

```diff
@@ -15,6 +15,15 @@
   </div>
 </div>
 
+<% if @section.body.present? %>
```

Comment:
I've taken the liberty to add this assertion to the test (see a42ca66c055ade6b2ce62dc040de04e35f3466d9) and force-pushed the branch.